pages/form-builder/contactus.tsx

```diff
@@ -0,0 +1,281 @@
+                    name={"email"}
+                    className="required w-[34rem]"
+                    required
+                  />
+                </div>
+                <fieldset className="focus-group">
+                  <legend className="gc-label required">
+                    {t("contactus.request.title")}{" "}
+                    <span data-testid="required" aria-hidden>
+                      ({t("required")})
+                    </span>
+                  </legend>
+                  <MultipleChoiceGroup
+                    name="request"
+                    type="radio"
+                    choicesProps={[
+                      {
+                        id: "request-question",
+                        name: "question",
+                        label: t("contactus.request.option1"),
+                        required: true,
+                      },
+                      {
+                        id: "request-feedback",
+                        name: "feedback",
+                        label: t("contactus.request.option2"),
+                        required: true,
+                      },
+                      {
+                        id: "request-demo",
+                        name: "demo",
+                        label: t("contactus.request.option3"),
+                        required: true,
+                      },
+                      {
+                        id: "request-other",
+                        name: "other",
+                        label: t("contactus.request.option4"),
+                        required: true,
+                      },
+                    ]}
+                  ></MultipleChoiceGroup>
+                </fieldset>
+                <div className="focus-group">
+                  <Label
+                    id={"label-description"}
+                    htmlFor={"description"}
+                    className="required"
+                    required
+                  >
+                    {t("contactus.description.title")}
+                  </Label>
+                  <Description id={"description-description"}>
+                    {t("contactus.description.description")}
+                  </Description>
+                  <TextArea
+                    id={"description"}
+                    name={"description"}
+                    className="required w-[34rem] mt-4"
+                    required
+                    characterCountMessages={{
+                      part1: t("formElements.characterCount.part1"),
+                      part2: t("formElements.characterCount.part2"),
+                      part1Error: t("formElements.characterCount.part1-error"),
+                      part2Error: t("formElements.characterCount.part2-error"),
+                    }}
+                  />
+                </div>
+                <div className="flex mt-14">
+                  <Button
+                    type="submit"
+                    className={` 
```

Comment:
So we have a style for this now. I know we are revising styling but probably better to use this one rather than repeat a bunch of tailwind classes everywhere and so to keep focus and hover states somewhat consistent 

https://github.com/cds-snc/platform-forms-client/blob/develop/pages/auth/login.tsx#L152

Reply:
No way tailwind classes are so hot right now! :) Heh I've been putting that decision off. To either go all in with the tailwind classes or use a combination of tailwind classes with custom classes.
I think we talked about this at some point with Bryan but I forget if we decided anything?